mofa/simulation/lammps.py

```diff
@@ -8,9 +14,103 @@ class LAMMPSRunner:
     Args:
         lammps_command: Command used to launch LAMMPS
     """
+    lammps_command = "npt_tri"
+    lmp_sims_root_path = "lmp_sims"
+    cif_files_root_path = "cif_files"
+    cif_files_paths = []
```

Comment:
These are superfluous because you define the values in `__init__` anyway. Plus, using a list as a default value [is problematic](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)